src/components/AddRepsModal/LiftInfo/index.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { LiftInfo } from "../../../types/LiftTypes";
+import Comment from "./Comment";
+import Links from "./Links";
+
+type Prop = {
+  liftInfo: LiftInfo;
+  onLiftInfoChange: (info: LiftInfo) => void;
+};
+
+type State = {
+  hasComment: boolean;
+};
+
+class LiftInfoContainer extends React.Component<Prop, State> {
+  constructor(props: Prop) {
+    super(props);
+    this.state = {
+      hasComment: !!props.liftInfo.comment
+    };
+  }
+  public render() {
+    return (
+      <div className="d-flex flex-column align-items-start">
+        <Comment
+          onCommentChange={this.onCommentChange}
+          commentValue={this.props.liftInfo.comment}
+          onCommentToggle={this.onCommentToggle}
+          hasComment={this.state.hasComment}
+        />
+        <Links
+          links={this.props.liftInfo.links}
+          onLinkAdd={this.onLinkAdd}
+          onLinkRemove={this.onLinkRemove}
+          onLinkChange={this.onLinkChange}
+        />
+      </div>
+    );
+  }
```

Comment:
And another one...
I am starting to wonder if there is a tslint rule for this...

Reply:
This one maybe?
"lines-between-class-methods"
https://github.com/eslint/eslint/issues/5949

Reply:
Seems like this rule has never made it into the `tslint`.

Reply:
Ok then.